pytorch_lightning/callbacks/device_stats_monitor.py

```diff
@@ -32,6 +33,10 @@ class DeviceStatsMonitor(Callback):
     Automatically monitors and logs device stats during training stage. ``DeviceStatsMonitor``
     is a special callback as it requires a ``logger`` to passed as argument to the ``Trainer``.
 
+    Args:
+        cpu_stats: if True, will also record CPU stats in addition to accelerator stats.
+            Default: ``True``.
```

Comment:
```suggestion
```

Reply:
For future reference, how come this change? I saw `trainer.py` use similar formatting